.github/workflows/build-release.yml

```diff
@@ -0,0 +1,75 @@
+name: Release
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+
+  release_amd64:
+    name: Release (AMD64 - CentOS 7 Target)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build code-server Binary
+        run: yarn && bash ./scripts/ci.bash    
+        env:
+           MAJOR_VERSION: '2'
+           TARGET: 'linux'
+           VERSION: '${MAJOR_VERSION}.${GITHUB_SHA}'
```

Comment:
Should we use the tag for the version? I'm thinking the hash is going to be sort of an eyesore as far as versions go.